cli/azd/pkg/auth/constants.go

```diff
@@ -0,0 +1,15 @@
+package auth
+
+import "fmt"
+
+const (
+	//#nosec G101 -- This is a false positive
+	remoteAuthCredName string = "RemoteCredential"
+
+	contentTypeHeader   string = "Content-Type"
+	authorizationHeader string = "Authorization"
+)
+
+func remoteCredentialError(err error) error {
```

Comment:
If we're buying into the idea that maybe the HTTP constants are not necessary, we could also just move the function `remoteCredentialError` into `remote_credential` and maybe remove `removeAuthCredName` entirely to keep things simple?

Reply:
It's a hard mental shift for me!!  to be honest, lol.  It might be a very solid paradigm in my head.  Somehow, I can't stop feeling observed by my inner me, giving me a look of `you're are not doing things right`.  

Even for such trivial things like "using a constant", instead of duplicating the same string in two files from the same namespace (I usually see packages as namespaces).  My mind is not letting me feel proud of myself by not using a constant, regardless of the specific scenario!!   I can't help thinking that, when duplicating strings, I am writing the code that I was writing when I was just in my first days as a developer. It feels like if I am not taking enough care of the code to try taking it from level 0 to at least level 1 (in a ranking that I would call Soft Eng 101).

Makes me wonder if there might be a limit in the path to removing abstractions/complexity/elegance where we could be going too far?  

Anyway, I bought the idea and removed the constants and move func to remote_credential :)  But I don't feel proud about it. LOL